Course_Part_04/src/Assignment13_SimpleCalculator.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Arrays;
+
+public class Assignment13_SimpleCalculator {
+
+    public static void main(String[] args) {
+
+        double a = 1;
+        double b = 3;
+        // Options: addition, subtraction, multiplication or division
+        calculatorPrint(a, b, "addition");
+        calculatorPrint(a, b, "subtraction");
+        calculatorPrint(a, b, "multiplication");
+        calculatorPrint(a, b, "division");
+        calculatorPrint(a, b, "multidivaddsub");
+
+    }
+
+    private static void calculatorPrint(double a, double b, String operator) {
+
+        String[] operators = {"addition", "subtraction", "multiplication", "division"};
+        if (Arrays.asList(operators).contains(operator)) {
```

Comment:
fancy! Maar is niet nodig, namelijk ...


de `calculator()` methode returned een Double in alle gevallen als de operator geldig is, anders returned ie `null`. Op basis hiervan kan de `if` bepalen welke `println` moet gebeuren.

Uitgewerkt:

```
private static void calculatorPrint(final double a, final double b, final String operator) {
    Double result = calculator(a, b, operator);
    if (result != null) {
	    System.out.printf("The %s of %f and %f is: %f%n",
			    operator,
			    a,
			    b,
			    result);
    } else {
	    System.out.printf("%s is not a valid operator!", operator);
    }
}
```